### ISSUE TITLE
More options for building images.

### ISSUE THREAD

**Issue by dev-gaur** [fabric8io/fabric8-maven-plugin#1534]
### Description
Currently, Docker and S2I are the only options for when building images. We need to add more alternatives.

- [ ] buildah
- [ ] jib

### Info
Buildah : (https://buildah.io/)
JIB : (https://github.com/GoogleContainerTools/jib)

**Comment by edeandrea** [comment]
I just wanted to add onto this thread. With RHEL 8 now out and dropping the docker tooling and instead bringing in buildah/podman, what is the direction for this plugin for support of that? I'd hate to see organizations update their CI/CD platforms and all of a sudden all their builds start failing because of this.

**Comment by dev-gaur** [comment]
Hi @edeandrea we're currently dealing with Jib integration to provide Docker less builds. Will start with Buildah integration short afterwards.

**Comment by edeandrea** [comment]
Just adding a gentle ping here so the stale-bot doesn't close this ticket.

**Comment by ctron** [comment]
Is there any update on this?

**Comment by dev-gaur** [comment]
Hi, we're gonna add new features on github.org/eclipse/jkube plugins.

https://developers.redhat.com/blog/2020/01/28/introduction-to-eclipse-jkube-java-tooling-for-kubernetes-and-red-hat-openshift/

JIB option has been added to FMP already and will be added soon to Jkube plugins. 

Although no concrete decision has been made, Kaniko integration is a big possibility for jkube,

**Comment by ctron** [comment]
I would be more interested in Podman/Buildah … as working with fabric8 based projects on Fedora 31 is currently not fun

**Comment by dev-gaur** [comment]
@ctron You can import the local tar archive of container image generated by JIB using `podman load -i image.tar`.

**Comment by ctron** [comment]
> @ctron You can import the local tar archive of container image generated by JIB using
> `podman load -i image.tar`.

I don't think that is an option. We have an upstream project, with an existing build system. I don't think anyone wants to change that, just because I am using Fedora.

**Comment by abhiroopghatak** [comment]
Latest release of org.eclipse.jkube plugins failing to create image on buildah , with error "no DOCKER_HOST environment variable, no read/writable '/var/run/docker.sock'".

Any support on buildah or any separate plugins ? please update!!

**Comment by manusa** [comment]
Support for this feature in Eclipse JKube is being tracked here https://github.com/eclipse/jkube/issues/66

I'm not sure about the status of Podman 2.0, but it may already be compatible with our current implementations.

Anyway, please follow-up on the provided [issue](https://github.com/eclipse/jkube/issues/66).
